dev/SearchBox.js

```diff
@@ -1,6 +1,9 @@
 import React, {Component, PropTypes as T} from 'react';
 import connect from '../src/connect.js';
 
-export default connect()(function({helper}) {
-  return <input onChange={(e) => helper.setQuery(e.target.value).search()} />;
+export default connect()(function({helper, search}) {
```

Comment:
I'm not against the idea for sure :). Just brainstorming: what about this was an option? Or a flag (even though can have their problems and we discussed them at length)? Do you think this behavior could be the default? is there a case when batching based on the next tick would not make sense?